### PR TITLE
Fix 404s for tablist/tabpanel in “ARIA: tab role”

### LINKED ISSUE #949: Issue with "ARIA: tab role": …
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

#### What information was incorrect, unhelpful, or incomplete?

Broken links to tablist and tabpanel

#### Specific section or headline?
First sentence

#### What did you expect to see?
Working links

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/tab_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/accessibility/aria/roles/tab_role/index.html
* Report started: 2021-01-05T07:30:47.857Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/949

### CODE REVIEW

**Review by Elchi3 — APPROVED** [review]

**Comment by sideshowbarker** [comment]
@ericwbailey, don’t want to merge this without your OK

**Comment by nschonni** [comment]
I fixed the missing `Tabpanel_Role` page in #1021, but I didn't see a page for "tablist" at the same time

**Review by ericwbailey — APPROVED** [review]

File: files/en-us/web/accessibility/aria/roles/tab_role/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 ---
 <div>{{Page("/en-US/docs/Web/Accessibility/ARIA/Roles", "Subnav")}}</div>
 
-<p><span class="seoSummary">The ARIA <code>tab</code> role indicates an interactive element inside a <a href="/en-US/docs/Web/Accessibility/ARIA/Roles/Tablist_Role">tablist</a> that, when activated, displays its associated <a href="/en-US/docs/Web/Accessibility/ARIA/Roles/Tabpanel_Role">tabpanel</a>.</span></p>
+<p><span class="seoSummary">The ARIA <code>tab</code> role indicates an interactive element inside a <code>tablist</code> that, when activated, displays its associated <code>tabpanel</code>.</span></p>
```
Comment (nschonni) [inline files/en-us/web/accessibility/aria/roles/tab_role/index.html:13]: Adding back the tabpanel page has been renamed. Currently it looks like it hasn't published yet https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tabpanel_Role
```suggestion
<p><span class="seoSummary">The ARIA <code>tab</code> role indicates an interactive element inside a <code>tablist</code> that, when activated, displays its associated <a href="/en-US/docs/Web/Accessibility/ARIA/Roles/Tabpanel_Role"><code>tabpanel</code></a>.</span></p>
```

Comment (sideshowbarker) [reply files/en-us/web/accessibility/aria/roles/tab_role/index.html]: > Adding back the tabpanel page has been renamed

Thanks much for fixing that

**Pull request merged**